atd-vze/src/scss/_custom.scss

```diff
@@ -1 +1,22 @@
 // Here you can add other styles
+
+div.fullscreen>div>button {
```

Comment:
I need someone else to verify, but I was able to see the default Mapbox buttons without these styles added here. If that can't be replicated, I think that this selector should be a little more specific because there are two different icons for the expand and collapse (depending on whether in full screen mode or not).

Reply:
Mike, I have been able to confirm that I am able to see the icons. I have no idea why they were so consistently not showing up for me before, but this time, I've tested by entirely wiping out my `node_modules` folder and my `packge-log.json` file and building from as scratch as I can conceive of doing. They seem to show up fine, so I am going to strike those changes from this PR.

Additionally, I have had to add two checks for null objects that end up containing data from the graphql-engine endpoint but cause a WSOD. I mention this here because I had had to add them when I first started working this issue, and then removed them because I found them not to be needed in testing, which feels a lot like this icon thing.

There's something going on, most likely on my computer or between my keyboard and chair that is preventing me from feeling like I have a 100% handle on what's going on here. That said, I will feel comfortable with shipping this PR with approval from you and others on the team.

Reply:
Hmmm I did initially see no icons before I npm installed so maybe that is a piece of it. Glad you sorted it out on your end. It looks great in the preview!